core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -770,5 +940,140 @@ $(document).ready(function () {
         }
       });
     });
+
+    var quantiles = "0,0.25,0.5,0.75,1.0";
+    $.getJSON(createRESTEndPointForExecutorsPeakMetricsSummariesPage(appId) + "?activeOnly=true&quantiles=" + quantiles,
```

Comment:
nit: can we have a smaller method name than this one? for eg: `createRESTEndPointForExecutorsSummaries` or some other smaller name?

Reply:
Done